app/src/androidTest/assets/features/create_profile.feature

```diff
@@ -54,9 +56,9 @@ Feature: Create a profile
 
     Examples:
      | name         | password  | repeatpassword    |
-     | Drake        | a         | b                 |
-     | CJ           | ^&*)(     | NetAnders         |
-     | Darth Vader  | 6782      | 4578              |
+     | Drake        | abcdefg   | bcdefgh           |
+     | CJ           | ^&*)(8999 | NetAnders         |
+     | Darth Vader  | 67824444  | 4578              |
```

Comment:
Since they were too short, the order of password faults would say that first, before them not being equal. This is more logical since you'd first want to create a correct password before entering the same twice.